shared/actions/fs.js

```diff
@@ -7,6 +7,27 @@ import * as Saga from '../util/saga'
 import * as Types from '../constants/types/fs'
 import {openInFileUISaga} from './fs-platform-specific'
 
+function* filePreview(action: FsGen.FilePreviewLoadPayload): Saga.SagaGenerator<any, any> {
+  const rootPath = action.payload.path
+
+  const dirent = yield Saga.call(RPCTypes.SimpleFSSimpleFSStatRpcPromise, {
+    path: {
+      PathType: RPCTypes.simpleFSPathType.kbfs,
+      kbfs: Constants.fsPathToRpcPathString(rootPath),
+    },
+  })
+
+  const meta = {
```

Comment:
These are already available from the folderList right? So when user clicks into the file preview, they should already be already in `state.fs.pathItems`. Did you do this as an intentional refresh to get the latest data?

Reply:
Yes. It might be somewhat old? (especially on desktop)